### PR TITLE
Nuget Bugfix

### PR DESCRIPTION
**Pull request by viceroypenguin**

### CODE REVIEW

File: Build/linq2db.LINQPad.nuspec [hunk 1]
```diff
@@ -12,7 +12,7 @@
 		<dependencies>
 			<group targetFramework=".NETCoreApp3.1">
 				<dependency id="LINQPad.Reference"                         version="1.3.0"     />
-				<dependency id="linq2db"                                   version="4.0.1"     />
+				<dependency id="linq2db"                                   version="4.1.0"     />
```
Comment (MaceWindu) [inline Build/linq2db.LINQPad.nuspec:15]: npgsql was also updated

Comment (MaceWindu) [reply Build/linq2db.LINQPad.nuspec:15]: and System.Text.Json

**Review by MaceWindu — CHANGES_REQUESTED** [review]
This could explain https://github.com/linq2db/linq2db.LINQPad/issues/72
I discarded https://github.com/linq2db/linq2db/issues/3610 as possible reason, but if we didn't releases 4.1 - it's probably the cause

**Review by viceroypenguin — COMMENTED** [review]
/sigh

File: Build/linq2db.LINQPad.nuspec (hunk 1 above)
Comment (viceroypenguin) [reply Build/linq2db.LINQPad.nuspec:15]: Thanks! One of these days I'll remember my head

**Pull request merged**